### PR TITLE
Fix #93: Convert Marketing tab content to Lesson Creation tab

### LINKED ISSUE #93: Convert Volunteer page's Marketing tab to Lesson Creation tab
- Rename files
- Rename tab from "Marketing" to "Lesson Creation"
- Swap out "Marketing" text content for that of "Lesson Creation".

> Oppia’s lesson creation team is responsible for creating a set of highly effective and enjoyable lessons that students around the world can use for free. The team follows a tested process to generate effective lessons that appeal to students from all backgrounds. Our goal is to not only teach students in an easily digestible way, but also encourage a positive attitude towards learning. Currently, we're focusing on basic mathematics, but intend to expand to other subjects in the future.

> What we do:
> - Break down math concepts into bite-sized lessons that build on each other
> - Come up with interesting stories for the lesson's "main plot"
> - Create scripts for lessons and implement them on Oppia
- Shuffle profiles: 
 1. - [ ] Move Diana profile to Development.
2. - [ ] Move Anmol and Dawson to Lesson Creation (Lesson Creation should have Joe, Anmol and Dawson)

### PR DESCRIPTION
**Pull request by hoangviet1993**
Fix #93 
- [x] Convert Marketing tab references to Lesson Creation tab.
- [x] Replace existing Marketing content with Lesson Creation.
- [x] Optimize Diana profile image (221kb -> 39kb)
- [x] Shuffle volunteer profiles: Diana is moved to "Development" and Lesson Creation now has Anmol, Joe and Dawson
- Changes in this PR are live at  https://oppia-foundation-test-server.appspot.com/volunteer @joeyannaphol , can you PTAL :D
- Some static screenshot of mobile and desktop view-width:
Desktop:
lesson_creation_tab_desktop_view
Mobile:
lesson_creation_tab_mobile_view

### CODE REVIEW

File: app/static/pages/volunteer/VolunteerInfoDefinitions.js [hunk 1]
```diff
@@ -97,19 +92,24 @@ oppiaFoundationWebsite.constant('VOLUNTEER_INFO', {
     position: 'Researcher and Lesson Creator'
   }],
 
-  marketing: [{
+  lesson_creation: [{
+    fullName: 'Anmol Shukla',
+    profilePictureImageFilename: 'anmol.jpg',
+    professionalQuote: 'I strongly identify with Oppia\'s objective of ' +
+    'providing free, enjoyable and accessible education to all.',
+    position: 'Developer, Machine Learning'
```
Comment (seanlip) [inline app/static/pages/volunteer/VolunteerInfoDefinitions.js]: Change to: "Developer and Lesson Creator"

File: app/static/pages/volunteer/tabs_template/lesson_creation_tab.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    </p>
+    <p>
+      Our goal is to not only teach
+      students in an easily digestible way, but also encourage a positive
+      attitude towards learning. Currently, we're focusing on basic mathematics,
+      but intend to expand to other subjects in the future.
+    </p>
+    <ng-include src="volunteerWorkDescriptionUrls.lessonCreation"></ng-include>
+    <md-button class="oppia-foundation-green-button"
+               href="https://goo.gl/forms/otv30JV3Ihv0dT3C3">
```
Comment (seanlip) [inline app/static/pages/volunteer/tabs_template/lesson_creation_tab.html]: Use this form instead: https://goo.gl/forms/w1JtEtRDpGMCk3aB3

File: app/static/pages/volunteer/VolunteerInfoDefinitions.js (hunk 1 above)
Comment (hoangviet1993) [reply app/static/pages/volunteer/VolunteerInfoDefinitions.js]: Done!

File: app/static/pages/volunteer/tabs_template/lesson_creation_tab.html (hunk 2 above)
Comment (hoangviet1993) [reply app/static/pages/volunteer/tabs_template/lesson_creation_tab.html]: Thanks for reminding me about this :D

**Pull request merged**